include/credits.h

```diff
@@ -0,0 +1,6 @@
+#ifndef GUARD_CREDITS_H
+#define GUARD_CREDITS_H
+
+
```

Comment:
?

Reply:
Created just in case but was never needed. Deleted.